### ISSUE TITLE
wget command to get env file in README is not working as expected

### ISSUE THREAD

**Issue by cchwala** [OpenSenseAction/OPENSENSE_sandbox#61]
This does not do the correct thing
```
wget https://github.com/OpenSenseAction/OPENSENSE_sandbox_environment/blob/main/environment.yml
```

**Comment by cchwala** [comment]
If `wget` is not available (e.g. on MacOS), this works
```
curl -O https://raw.githubusercontent.com/OpenSenseAction/OPENSENSE_sandbox_environment/main/environment.yml
```

**Comment by cchwala** [comment]
As suggested by @fenclmar, we should also write about the most simple option, to download the file via the browser and just make sure that you are in the same dir when running the env creation command.

**Comment by cchwala** [comment]
closed via #65
